server/filters/generate-id.js

```diff
@@ -0,0 +1,5 @@
+import shortid from 'shortid';
+
+export default function generateId(blank) {
+  return shortid.generate();
+};
```

Comment:
Why can we not use the image id?

Reply:
er, yeah we can... not sure why I did this now